spec/controllers/idv/how_to_verify_controller_spec.rb

```diff
@@ -238,7 +238,7 @@
     end
 
     context 'undo/back' do
-      it 'sets skip_doc_auth to nil and does not redirect' do
+      it 'sets skip_doc_auth_from_how_to_verify to nil and does not redirect' do
```

Comment:
This test is currently being rewritten by @n1zyy over [here](https://github.com/18F/identity-idp/commit/58fa3b873d522309e6ae33642c65d2cb203cc81f)